salt/client/mixins.py

```diff
@@ -390,7 +393,10 @@ def _low(self, fun, low, print_event=True, full_return=False):
                 try:
                     data['return'] = func(*args, **kwargs)
                 except TypeError as exc:
-                    data['return'] = '\nPassed invalid arguments: {0}\n\nUsage:\n{1}'.format(exc, func.__doc__)
+                    data['return'] = salt.utils.text.cli_info('Usage:\n{1}'.format(exc, func.__doc__),
```

Comment:
We're missing a `{0}` here, I think? We've got two in .format, but only one `{}` string.

Reply:
Thank the linter for spotting it! Hopefully it'll be better next time!